NuRadioReco/framework/station.py

```diff
@@ -188,51 +188,62 @@ def get_reference_reconstruction(self):
     def get_reference_direction(self):
         if self.__reference_reconstruction == 'RD':
             return self.get_parameter('zenith'), self.get_parameter('azimuth')
-        if self.__reference_reconstruction == 'MC':
+        elif self.__reference_reconstruction == 'MC':
             return (
-                self.get_sim_station().get_parameter('zenith'), 
+                self.get_sim_station().get_parameter('zenith'),
                 self.get_sim_station().get_parameter('azimuth')
             )
+        else:
+            logger.error(f"Reference reconstruction not set / unknown: {self.__reference_reconstruction}")
+            raise ValueError(f"Reference reconstruction not set / unknown: {self.__reference_reconstruction}")
+
 
     def get_magnetic_field_vector(self, time=None):
-        if(self.__reference_reconstruction == 'MC'):
+        if self.__reference_reconstruction == 'MC':
             return self.get_sim_station().get_magnetic_field_vector()
-        if(self.__reference_reconstruction == 'RD'):
-            if time is not None:
-                logger.warning("time dependent magnetic field model not yet implemented, "
-                               "returning static magnetic field for the ARIANNA site")
-            from radiotools import helper
-            return helper.get_magnetic_field_vector('arianna')
+        elif self.__reference_reconstruction == 'RD':
+            logger.error(
+                "Magnetic field for `self.__reference_reconstruction == 'RD'` not implemented yet. "
+                "Please use `radiotools.helpers.get_magnetic_field_vector(site)` for the site you are interested in.")
+            raise NotImplementedError(
+                "Magnetic field for `self.__reference_reconstruction == 'RD'` not implemented yet. "
+                "Please use `radiotools.helpers.get_magnetic_field_vector(site)` for the site you are interested in."
+            )
+        else:
+            logger.error(f"Reference reconstruction not set / unknown: {self.__reference_reconstruction}")
+            raise ValueError(f"Reference reconstruction not set / unknown: {self.__reference_reconstruction}")
```

Comment:
Technically breaks backwards compatibility but I agree this should be changed. Only I think the module is called `radiotools.helper` and not `helpers`?